src/vsr/client.zig

```diff
@@ -292,11 +292,10 @@ pub fn Client(comptime StateMachine_: type, comptime MessageBus: type) type {
         }
 
         /// Releases a message back to the message bus.
-        pub fn unref(self: *Self, message: *Message) void {
```

Comment:
Let's adjust the docs for get_message to say that the message must end up in release or request, and that it must _not_ be `ref`ed, as this flow is different from how one typically uses the `Message`. "`.ref()` if you store the message" is the general rule we use elsewhere, and here we are deliberately violating that.